mock_service/lib/behavior.py

```diff
@@ -21,6 +21,17 @@
 from lib.types import Event, ExecutionContext
 
 log = logging.getLogger("behavior")
+log.setLevel(logging.DEBUG)
+
+# Create a file handler and set the log file name
+file_handler = logging.FileHandler("mock_service.log")
```

Comment:
Do we really want to set this only for behavior.py?

Reply:
seemed to me to have the best output but yeah maybe could add it to the other ones too.

Reply:
I think for the other ones logging in log file is not necessary. Did this for seeing logs in the GUI. I remember now.

Reply:
Long term I think it might be confusing that just `behavior.py` is writing to `mock_service.log`. Maybe you want to create this file in a central location and only let behavior.py write to it with default `INFO` level?

If it serves a purpose, it seems OK for now.